app/views/posts/index.html.erb

```diff
@@ -1,3 +1,10 @@
-<h1>List of Posts</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/posts/index.html.erb</p>
+<%= render partial: 'users/user', locals: { user: @user } %>
+<% @posts.each do |post| %>
+  <%= render partial: 'posts/post_with_comments', locals: { post: post } %>
+<% end %>
+
+<% if @user.present? %>
+  <%= link_to 'Pagination', user_path(@user) %>
+<% else %>
+  <p>No posts available for pagination.</p>
```

Comment:
- [ ] Great job working on the views so far but I think it would be great if you made like the given wireframe since you should at least make the same layout so kindly adjust your element so you have the image for the user and his/her name in a box beside the image then a box contains the first post and then then a box contains the comments with username beside it, please make them the same given wireframe 🔥💪🏻🎉


![image](https://github.com/richie1988/blog-app/assets/58553711/ead20f39-8e2f-4e77-b06f-e530d1fe39b7)


- Your
![image](https://github.com/richie1988/blog-app/assets/58553711/8939aac9-7dda-4e9b-b4c2-dec05bb2fc67)



- Given

![image](https://github.com/richie1988/blog-app/assets/58553711/9de88c6a-8af7-4f57-99bd-a02008617434)